internal/dag/httpproxy_processor.go

```diff
@@ -993,8 +993,18 @@ func (p *HTTPProxyProcessor) computeRoutes(
 				return nil
 			}
 			if service.Mirror {
-				r.MirrorPolicy = &MirrorPolicy{
-					Cluster: c,
+				// Default value if weight is omitted is 0. Setting this to 100 to retain backward-compat behavior.
+				// Edge case is a user intentionally/explicitly setting the Weight to 0 (ie. disabling the mirror).
```

Comment:
Let's add some godoc to `apis/v1/projectcontour/httpproxy.go` on the Weight/Mirror fields documenting this behavior, along with the "normal" weighted mirror behavior, for users.

You'll need to run `make generate` and commit the results afterwards.

Reply:
Updated and generated docs